src/compressed_tensors/transform/transform_scheme.py

```diff
@@ -34,10 +36,13 @@ class TransformScheme(BaseModel):
     :param randomize: True if uniquely randomized transform weights should be used,
         otherwise use identical transform weights where applicable
     :param requires_grad: True if weights include gradients for training
+    :param precision: Precision at which this transform should be applied. This applies
+        to both weight fusing and online rotations
     """
 
     type: str
     apply: List[TransformArgs] = Field(default_factory=list)
     randomize: bool = Field(default=False)
     requires_grad: bool = Field(default=False)
     head_dim: Optional[int] = Field(default=None)
+    precision: TorchDtype = Field(default=torch.float32)
```

Comment:
I think if we want to expose this instead of hard-coding, we should default to float64 for fused. the application will only happen once per layer, so it's not much overhead, and that's how original implementations are set up.

Reply:
As noted in the docs, this precision also affects online rotations. I found a considerable runtime difference going from float32 to float64 for quip, so I think float32 should be fine? This precision will be overridden by the modifier anyway.

Reply:
that's not a concern for fused though. we'd always want float64, i'm not sure we want to expose that to the user. we could treat precision as only the precision for online, and always promote to float64 when applying transform for fused?

Reply:
That seems fair